src/.eslintrc.cjs

```diff
@@ -22,52 +23,54 @@ module.exports = {
         "quotes": [ "error", "double" ],
         "semi": [ "error", "always" ],
         "camelcase": "off",
-        // "no-unused-vars": "off",
         "eqeqeq": "off",
```

Comment:
Remove this rule, please

Reply:
`"no-unused-vars": "off"` is removed
notice: but we have it in `IMA` and in `network browser`